tests/alias_test.py

```diff
@@ -0,0 +1,15 @@
+import pydantic
+
+
+class TestModel(pydantic.BaseModel):
+    some_attribute: str = pydantic.Field(alias="some-attribute")
+
+    class Config:
+        allow_population_by_field_name = True
+
+
+tm1 = TestModel(some_attribute="test1")
+print(tm1)
```

Comment:
Remove prints

Reply:
Ah sorry for that, this was just a quick test for setting attributes with alias. This was not meant for upload, I'll correct it.